src/preload/syscall_hook.S

```diff
@@ -1,4 +1,8 @@
+#if defined(__aarch64__)
+        .set preload_thread_locals,0x70010000
+#else
         .set preload_thread_locals,0x70001000
+#endif
```

Comment:
Why doesn't this use `0x70000000 + RR_PAGE_SIZE`?

Reply:
This one was actually just because I was too lazy to check if the macro was defined by this point ;-p...